deploy/kubeturbo/templates/configmap.yaml

```diff
@@ -55,4 +55,10 @@ data:
         "workloadController": "{{ default "" .Values.annotationWhitelist.workloadController }}"
       {{- end }}
       }
+    }
+  turbo-dynamic.config: |-
```

Comment:
I wonder if it will be better to call this `turbo-autoreload.config`?

Reply:
Change it to `autoreload`, and also updated the related variable's name and logging message.  Thanks for the review @ading1977